src/lib/cli/format.js

```diff
@@ -122,14 +122,27 @@ export function requoteArgs( args: Array<string> ): Array<string> {
 			return arg.replace( /^--(.*)=(.*)$/, '--$1="$2"' );
 		}
 
-		if ( arg.includes( ' ' ) ) {
+		if ( arg.includes( ' ' ) && ! isJsonObject( arg ) ) {
 			return `"${ arg }"`;
 		}
 
 		return arg;
 	} );
 }
 
+export function isJsonObject( str: string ): boolean {
+	return typeof str === 'string' && str.trim().startsWith( '{' ) && isJson( str );
```

Comment:
Is doing a `typeof str` here necessary / helpful? This isn't an application boundary where we're checking something that can't be statically verified, and we've already done a `.includes` on this value before it is passed to this function.